src/core/public/chrome/chrome_service.test.ts

```diff
@@ -177,33 +175,37 @@ describe('start', () => {
 
       await expect(promise).resolves.toMatchInlineSnapshot(`
         Array [
-          true,
-          true,
           false,
-          true,
+          false,
+          false,
+          false,
         ]
       `);
     });
 
-    it('always emits false if embed query string is preset when set up', async () => {
+    it('emits false until manually overridden when in embed mode', async () => {
       window.history.pushState(undefined, '', '#/home?a=b&embed=true');
+      const startDeps = defaultStartDeps([new FakeApp('alpha')]);
+      const { navigateToApp } = startDeps.application;
+      const { chrome, service } = await start({ startDeps });
```

Comment:
This test was misleading and incomplete. the value would have changed to true (both in master and PR version) if both these actions were performed in any order:
- we navigate to an app
- we manually set chrome visibility using `chrome.setIsVisible(true)`